src/eterna/mode/PoseEdit/PoseEditMode.ts

```diff
@@ -132,7 +134,7 @@ export type SubmitSolutionData = {
     'pointsrank-before'?: RankScrollData | null;
     'pointsrank-after'?: RankScrollData | null;
     'selected-nts'?: number[];
-    'annotations'?: AnnotationDataBundle;
+    'annotations'?: string;
```

Comment:
Wanted to just check-in on the motivation behind your preference for using string here. Definitely out of curiosity than criticism

Reply:
It needs to be stringified when included as a post parameter. If I use an object directly, it will just toString() it and it winds up as "[Object object]"